src/components/editor/EditAuthorsForm.jsx

```diff
@@ -8,6 +8,28 @@ import _ from 'lodash';
 // material-ui
 import Chip from 'material-ui/Chip';
 
+class AuthorChip extends BaseComponent {
```

Comment:
Hmm... I think it's actually a bit nicer having it here as it's just a wrapper to avoid the performance costs of creating a new function every time that is fixed like [this](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md#lists-of-items). And it could always be modularized out if we reused that component somewhere though it's very specific so I doubt it. Since the file is so small even with the two of them I felt it was easier to read instead of having to go to a new file just for a small wrapper.
What do you say?